meshroom/core/desc.py

```diff
@@ -44,6 +44,15 @@ def validateValue(self, value):
         """
         raise NotImplementedError("Attribute.validateValue is an abstract function that should be implemented in the derived class.")
 
+    def validateValueType(self):
```

Comment:
In fact, the function name is very similar to `def validateValue(self, value)` which does something very different. It would be good to find another name to avoid the ambiguity between the two.

Maybe `def checkValueTypes(self)`.

Reply:
Done using "checkValueTypes" (I didn't have a better name idea in store 🙃).